dolphinscheduler-tools/src/main/bin/demo-start.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
What about renaming this file to something like `create-demo-processes.sh`? `-start` sounds like it's a long-running server but it is not

Reply:
Has been modified